src/main/java/module/TicTacToeService.java

```diff
@@ -0,0 +1,157 @@
+package main.java.module;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+public class TicTacToeService {
+    TicTacToeBoard ticTacToeBoard = new TicTacToeBoard();
+    Queue<Player> playerSequence;
```

Comment:
If you're not going to remove any of the pieces then this should be a List and not a Queue